services/src/main/java/org/keycloak/exportimport/util/ImportUtils.java

```diff
@@ -261,7 +262,9 @@ public static void importFederatedUsersFromStream(KeycloakSession session, Strin
     private static void importUsers(KeycloakSession session, RealmProvider model, String realmName, List<UserRepresentation> userReps) {
         RealmModel realm = model.getRealmByName(realmName);
         for (UserRepresentation user : userReps) {
-            RepresentationToModel.createUser(session, realm, user);
+            if (!user.getUsername().startsWith(ServiceAccountConstants.SERVICE_ACCOUNT_USER_PREFIX)) {
```

Comment:
I am not 100% sure, but does this mean that serviceAccount users will be re-created after export/import? This will mean different "sub" in the tokens for same service-account after export/import. It is probably ok, so approving this. Leaving to you to decide if you come with any corner case where this behaviour can cause issues.

Reply:
In fact, service account users will never be created but only user accounts. The reason for that if statement is that service accounts should not be created when creating users in an LDAP server, for instance.

Reply:
ah ok. I've approved already. Leaving the merge to you.